lua/bufferline/render.lua

```diff
@@ -131,14 +134,16 @@ local function render()
       icon =
         (letter ~= nil and letter or ' ') ..
         (has_icons and ' ' or '')
-    elseif has_icons then
+    else
```

Comment:
Changed into an `else` where the individual settings `has_numbers` and `has_icons` are independently evaluated and placed onto the bufferline. This allows both settings to exist at once.